api/src/main/java/com/codeforcommunity/rest/subrouter/authenticated/ProtectedSchoolRouter.java

```diff
@@ -326,8 +327,8 @@ private void handleCreateBookLog(RoutingContext ctx) {
     UpsertBookLogRequest request =
         RestFunctions.getJsonBodyAsClass(ctx, UpsertBookLogRequest.class);
     int schoolId = RestFunctions.getPathParamAsInt(ctx, "school_id");
-    processor.createBookLog(userData, schoolId, request);
-    end(ctx.response(), 201);
+    BookLog log = processor.createBookLog(userData, schoolId, request);
+    end(ctx.response(), 200, JsonObject.mapFrom(log).toString());
```

Comment:
Why did we change this from `201` to `200` if we're creating a new log entry?

Reply:
Fixed this, it was my bad